src/Commands/SyncFile.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Spatie\UptimeMonitor\Commands;
+
+use Spatie\UptimeMonitor\Models\Monitor;
+use Spatie\UptimeMonitor\Exceptions\CannotSaveMonitor;
+
+class SyncFile extends BaseCommand
+{
+    protected $signature = 'monitor:sync-file
+                            {path : Path to JSON file with monitors}
+                            {--delete-missing : Delete monitors from the database if they\'re not found in the monitors file}';
+
+    protected $description = 'One way sync monitors from JSON file to database';
+
+    public function handle()
+    {
+        $json = file_get_contents($this->argument('path'));
+
+        $monitorsInFile = collect(json_decode($json, true));
+
+        $this->validateMonitors($monitorsInFile);
+
+        $this->createOrUpdateMonitorsFromFile($monitorsInFile);
+
+        $this->deleteMissingMonitors($monitorsInFile);
+    }
+
+    protected function validateMonitors($monitorsInFile)
+    {
+        $monitorsInFile
+            ->filter(function ($monitorAttributes) {
+                return empty(preg_match('/^http:\/\/|^https:\/\//', $monitorAttributes['url']));
+            })
+            ->first(function ($invalidMonitorAttributes) {
```

Comment:
It's a bit overkill to use a collection function here. Replace this with a regular `if` statement.